simtools/corsika/corsika_output.py

```diff
@@ -926,10 +1019,9 @@ def num_photons_per_telescope(self):
         numpy.array
             Number of photons per telescope.
         """
-        if self._num_photons_per_telescope is None:
-            self._num_photons_per_telescope = np.sum(
-                np.array(self.num_photons_per_event_per_telescope), axis=0
-            )
+        self._num_photons_per_telescope = np.sum(
```

Comment:
because I want to allow the user to change the main flags `individual_telescopes` and `telescope_indices` and run `set_histograms` again. in that case, `_num_photons_per_telescope ` has to be recalculated to consider the possibly different telescopes given by `telescope_indices`

Reply:
OK!